docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -110,7 +107,7 @@ Object colors can vary dramatically based on the light source.
 We recommend you verify the desired color detection value under actual lighting conditions.
 To determine the color value from the actual cam component image, you can use a pixel color tool, like [Color Picker for Chrome](https://chrome.google.com/webstore/detail/color-picker-for-chrome/clldacgmdnnanihiibdgemajcfkmfhia).
 
-If the color is not reliably detected, drag the **Hue Tolerance** slider to the right or navigate to the **Raw JSON** tab to increase the `hue_tolerance_pct`.
+If the color is not reliably detected, drag the **Hue Tolerance** slider to the right or switch to **JSON** mode to increase the `hue_tolerance_pct`.
```

Comment:
Maybe point out that they can't test this right now? I think that would be a good addition 
```suggestion
In the following steps you'll be able to test the color detector.
If you notice the color is not reliably detected, drag the **Hue Tolerance** slider to the right or switch to **JSON** mode to increase the `hue_tolerance_pct`.
```